automation/docker/run-tests.sh

```diff
@@ -80,7 +80,7 @@ if [ "$DOCKERHOST" != "" ]; then
     HOST_MAPPING="--add-host=firecloud-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=firecloud-orchestration-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=rawls-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=thurloe-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} --add-host=sam-fiab.dsde-${ENV}.broadinstitute.org:${DOCKERHOST} -e SLACK_API_TOKEN=$SLACK_API_TOKEN -e BUILD_NUMBER=$BUILD_NUMBER -e SLACK_CHANNEL=${SLACK_CHANNEL}"
 fi
 
-TEST_ENTRYPOINT="testOnly -- -l ProdTest"
+TEST_ENTRYPOINT="testOnly -- -n GooglePassing"
```

Comment:
Why inclusive instead of exclusive? I think it's better to default to including new tests rather than having to remember to explicitly include them.

Reply:
yah this was just temporary.